telemetry/main.schema.json

```diff
@@ -33,6 +33,10 @@
           "type": "string",
           "pattern": "^\\d{2}\\."
         },
+        "displayVersion": {
+          "type": "string",
+          "pattern": "^\\d{2}\\."
```

Comment:
We can bump it to 3 digits, but i wouldn't go restrict it further right now.
The current main goal seems to be to match the existing data, tightening the schema heavily is something we can look at later (i want this for client test validation etc. anyway).